tests/__init__.py

```diff
@@ -9,6 +9,7 @@
 
 SAMPLE_DIRECTORY = "test_docs"
 BASIC_SAMPLE = "sample-pdf.pdf"
+BASIC_SAMPLE2 = "sample-doc.doc"
```

Comment:
Ideally, I'd like to not have variables with a number appended to them, as it's not easy to reason about their purpose. Since it's used in a single test, I'd propose moving this logic there.

Reply:
I agree. I renamed `sample_doc` to `sample_pdf` and `sample_doc2` to `sample_doc`. I sneaked in another commit for passing `sample_pdf` as a fixture instead of as a TestBase.sample_doc as well. I hope you don't mind.